<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pwf2int</class>
 <widget class="QDialog" name="pwf2int">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>181</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>411</width>
    <height>181</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>411</width>
    <height>181</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>pwf2int</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pwf2main.qrc">
    <normaloff>:/img/pwf2tools.png</normaloff>:/img/pwf2tools.png</iconset>
  </property>
  <widget class="QTabWidget" name="intTabs">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>411</width>
     <height>181</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="extractTab">
    <attribute name="title">
     <string>Extract an INT</string>
    </attribute>
    <widget class="QPushButton" name="extractOpenInt">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>30</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Open an INT!</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="extractIntPath">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>281</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="extractFindIntLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>391</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Find your INT file!</string>
     </property>
    </widget>
    <widget class="QLabel" name="extractFindFolderLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>391</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Find a folder to extract to!</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="extractFolderPath">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>281</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="extractOpenFolder">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>90</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Find a folder!</string>
     </property>
    </widget>
    <widget class="QPushButton" name="extractRun">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>120</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Extract your INT!</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="createTab">
    <attribute name="title">
     <string>Create an INT</string>
    </attribute>
    <widget class="QLabel" name="createSaveIntLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>391</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Save your INT!</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="createIntPath">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>281</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="createSaveInt">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>30</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Save an INT!</string>
     </property>
    </widget>
    <widget class="QLabel" name="createFindFolderLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>391</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Find your folder with your modified INT content!</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="createFolderPath">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>281</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="createOpenFolder">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>90</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Find a folder!</string>
     </property>
    </widget>
    <widget class="QPushButton" name="createRun">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>120</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Create an INT!</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="pwf2main.qrc"/>
 </resources>
 <connections/>
</ui>
